unique_in_order.py

```diff
@@ -0,0 +1,12 @@
+def unique_in_order(sequence):
+    if len(sequence) == 0:
+        return []
```

Comment:
This is call a _guard clause_.  Under a certain condition just return without bothering to have the logic below handle the one off.  It's a good design pattern for case like this when it simplifies the logic of the function.